packages/utilities/src/styled.tsx

```diff
@@ -30,6 +30,11 @@ export type StyleFunction<TStyleProps, TStyleSet> = IStyleFunctionOrObject<TStyl
   __noStyleOverride__: boolean;
 };
 
+function useForceUpdate() {
```

Comment:
we have this hook already in `react-hooks` package

Reply:
Yes, but `@uifabric/react-hooks` depends on `@uifabric/utilities`, so I can't create a circular dependency.

Reply:
ah that sucks